tensorflow-core/tensorflow-core-api/src/main/java/org/tensorflow/internal/c_api/presets/tensorflow.java

```diff
@@ -385,6 +460,79 @@ public void map(InfoMap infoMap) {
+                .pointerTypes("BytePointer"))
+        .put(new Info("absl::Span", "tensorflow::gtl::ArraySlice").annotations("@Span"))
+        .put(
+            new Info("std::vector<tensorflow::Output>").pointerTypes("NativeOutputVector").define())
+        .put(new Info("tensorflow::Output").javaNames("NativeOutput"))
+        .put(new Info("tensorflow::Operation").javaNames("NativeOperation"))
+        .put(new Info("tensorflow::Status").javaNames("NativeStatus").purify())
+        .put(
+            new Info("tensorflow::int32")
+                .cast()
+                .valueTypes("int")
+                .pointerTypes("IntPointer", "IntBuffer", "int[]"))
+        .put(
+            new Info(
+                    "tensorflow::CompositeOpScopes",
+                    "tensorflow::Input",
+                    "tensorflow::InputList",
+                    "tensorflow::OutputHash",
+                    "tensorflow::StackFrame",
+                    "tensorflow::StatusGroup",
+                    "tensorflow::internal::TF_StatusDeleter",
+                    "tensorflow::GraphDef",
+                    "tensorflow::Scope::graph_as_shared_ptr",
+                    "tensorflow::Scope::ToGraphDef",
+                    "tensorflow::Scope::ToGraph",
+                    "tensorflow::Scope::DoShapeInference",
+                    "tensorflow::Scope::DisabledShapeInferenceScope",
+                    "tensorflow::Scope::control_deps",
+                    "tensorflow::Scope::WithKernelLabel",
+                    "tensorflow::Scope::ClearColocation",
+                    "tensorflow::Scope::ColocateWith",
+                    "tensorflow::Scope::ColocateWith",
+                    "tensorflow::Scope::WithXlaCluster",
+                    "tensorflow::Scope::WithAssignedDevice",
+                    "tensorflow::Scope::status",
+                    "tensorflow::Scope::UpdateStatus",
+                    "tensorflow::Status::code",
+                    "tensorflow::CreateOutputWithScope",
+                    "TF_OperationDescription::colocation_constraints",
+                    "tensorflow::Operation::num_inputs",
+                    "tensorflow::Operation::input_type",
+                    "tensorflow::Operation::input",
+                    "tensorflow::Operation::num_outputs",
+                    "tensorflow::Operation::output_type",
+                    "tensorflow::Operation::output",
+                    "tensorflow::Operation::hash",
+                    "tensorflow::Output::hash",
+                    "tensorflow::Output::type",
+                    "tensorflow::Status::GetAllPayloads",
+                    "tensorflow::Status::ReplaceAllPayloads",
+                    "tensorflow::Status::ErasePayload",
+                    "tensorflow::Status::SetPayload",
+                    "tensorflow::Status::GetPayload",
+                    "tensorflow::Node::SetStackTrace",
+                    "tensorflow::Node::GetStackTrace")
```

Comment:
Yeah, we would, and while it's not great I don't see any alternatives. I'm not sure how much work @saudet has to do for TF upgrades now, but there is already enough stuff that might break I don't think checking this will add that much.

Reply:
AFAIK, this is all part of the deprecated C++ API, right? If that's the case, if anything changes it's probably just going to disappear at some point...